src/box/memtx_tx.c

```diff
@@ -1932,6 +1958,24 @@ memtx_tx_tuple_clarify_impl(struct txn *txn, struct space *space,
 	return result;
 }
 
+/**
+ * Helper of @sa memtx_tx_tuple_clarify.
+ * Detect and return is_prepared_ok flag.
+ */
+static bool
+detect_whether_prepared_ok(struct txn *txn)
+{
+	/*
+	 * The best effort that we can make for determining isolation level
+	 * is that decide is that a read-only transaction or not. For read
+	 * only (including autocommit select, that is txn == NULL) we should
+	 * choose READ_CONFIRMED that does not see prepared tuples in order
```

Comment:
Ditto: it's hard to understand what you mean by saying
> we should choose READ_CONFIRMED that does not see prepared tuples in order to avoid dirty read.

I.e., how can an isolation level _see_ something? Perhaps, this would sound better:
```suggestion
	 * choose READ_CONFIRMED so the transaction does not see prepared tuples in order
```